examples/models/sklearn_iris/sklearn_iris.ipynb

```diff
@@ -124,7 +124,7 @@
     "Then port-forward to that ingress on localhost:8003 in a separate terminal either with:\n",
     "\n",
     " * Ambassador: `kubectl port-forward $(kubectl get pods -n seldon -l app.kubernetes.io/name=ambassador -o jsonpath='{.items[0].metadata.name}') -n seldon 8003:8080`\n",
-    " * Istio: `kubectl port-forward $(kubectl get pods -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') -n istio-system 8003:80`"
+    " * Istio: `kubectl port-forward $(kubectl get pods -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') -n istio-system 8003:8080`"
```

Comment:
I'm confused by this. Tried port-forwarding to 8080 on my istio install and it doesn't work. The pod has a port 80 and not an 8080 for me. (Checked with `kubectl get pod istio-ingressgateway-589ff776dd-6fkd2 -n istio-system -o jsonpath='{.spec.containers[0].ports}'`.) I'm looking at an istio 1.1 install.

Reply:
@kinihun could you double-check?

Reply:
Hi @ryandawsonuk ,
I am using Istio version 1.6.8, and it exposes the following ports.
I tested with 8443 because I have TLS certificate installed so I can confirm that 8443 works.
`  ports:
  - name: https
    port: 8443
    protocol: TCP
  - name: tls
    port: 15443
    protocol: TCP
  - name: status-port
    port: 15021
    protocol: TCP
  - name: http2
    port: 8080
    protocol: TCP`

Reply:
/assign @axsaucedo

Reply:
@kinihun does `kubectl port-forward -n istio-system svc/istio-ingressgateway 8003:80` work for you? wondering if we can get something that works across versions

Reply:
@ryandawsonuk mapping to port 80 does not work for me. Mapping to port 8080 does.

Reply:
For me `kubectl port-forward -n istio-system svc/istio-ingressgateway 8080:80` works with istio 1.6.0